src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -559,5 +559,9 @@ void FrontEnd::add_extension(const std::shared_ptr<ov::Extension>& extension) {
                    std::dynamic_pointer_cast<ov::frontend::tensorflow::ConversionExtension>(extension)) {
         m_conversion_extensions.push_back(tensorflow_conv_ext);
         m_op_translators[tensorflow_conv_ext->get_op_type()] = tensorflow_conv_ext->get_converter();
+    } else if (auto op_base_ext = std::dynamic_pointer_cast<ov::BaseOpExtension>(extension)) {
+        for (const auto& attached_ext : op_base_ext->get_attached_extensions()) {
+            add_extension(attached_ext);
```

Comment:
Why just we update Op extension? I think we should also update `m_op_translators` dictionary.

Reply:
I don't think so, we extract `ConversionExtension` from `OpExtension` and add it using `add_extension` so we do not need to do anything extra.